packages/components/tests/dummy/app/templates/components/reveal.hbs

```diff
@@ -0,0 +1,163 @@
+      </Hds::Reveal>
+    </SF.Item>
+
+    <SF.Item @label="With HTML formatted content (containing alert & list)">
+      <Hds::Reveal @text="Show me how">
+        <Hds::Alert @type="compact" as |A|>
+          <A.Description>Instructions to add the verification record to your domain host</A.Description>
+        </Hds::Alert>
+        <ol>
+          <li>
+            For the record type, select
+            <b>TXT</b>.
+          </li>
+          <li>
+            In the
+            <b>Name/Host/Alias</b>
+            field, enter
+            <b>@</b>
+            or leave blank. Your host might require you to enter your domain, which looks like
+            <b>example.com</b>
+          </li>
+          <li>
+            In the
+            <b>Time to Live (TTL)</b>
+            field, enter
+            <b>86400</b>
+            or leave the default.
+          </li>
+          <li>
+            In the
+            <b>Value/Answer/Destination</b>
+            field, paste the verification record you copied.
+          </li>
+          <li>
+            Save the record.
+          </li>
+        </ol>
+      </Hds::Reveal>
+    </SF.Item>
+
+    <SF.Item @label="With interactive content">
+      <Hds::Reveal @text="More options">
+        See here for more info:
+        <a href="https://www.hashicorp.com/" target="_blank" rel="noopener noreferrer">HashiCorp</a>
+      </Hds::Reveal>
+    </SF.Item>
+  </Shw::Flex>
+
+  <Shw::Text::H3>Used next to other content</Shw::Text::H3>
+
+  <Shw::Flex @direction="column" {{style gap="2rem"}} as |SF|>
+    <SF.Item @label="Used beneath a form input">
+      <Hds::Form::TextInput::Field as |F|>
+        <F.Label>Private subnet 1</F.Label>
+        <F.HelperText>Ensure that the private subnets have access to the external network.
+        </F.HelperText>
+      </Hds::Form::TextInput::Field>
+      <Hds::Reveal @text="Where can I find this?">
+        <Hds::Alert @type="compact" as |A|>
+          <A.Description>Navigate to Subnets(1), if you already have subnet created for the VPC of choice, copy the
+            Subnet ID(2). Otherwise, Create subnet(3) for chosen VPC.</A.Description>
+        </Hds::Alert>
+        <p>
+          <img
+            src="https://portal.cloud.hashicorp.com/engines-dist/consul/images/aws-subnet-instruction-7c866fcfb753c4794ff07afe5523818c.png"
+            alt="example"
+          />
+        </p>
+      </Hds::Reveal>
+    </SF.Item>
+
+    <SF.Item @label="With text above and below">
+      <p class="hds-typography-body-300">
+        Lorem ipsum dolor sit amet consectetur
+      </p>
+      <Hds::Reveal @text="More options">
+        Additional content
+      </Hds::Reveal>
+      <p class="hds-typography-body-300">
+        adipisicing elit. Doloremque blanditiis sapiente iste beatae voluptates voluptatum.
+      </p>
+    </SF.Item>
+
+    <SF.Item @label="Used inline with text (edge case)">
```

Comment:
I would think that we want to explicitly not include this, maybe even have documentation for this as a "don't".

Reply:
I included it partially just to see how content right next to it looks in relation although I don't think it would ever be practically used this way. Maybe I could move it to another more clear "Edge case" section? 

@didoo @alex-ju What do you think? Would it be helpful to keep this example around or should I just delete it?

Reply:
This has since been deleted so I'm resolving this thread, thanks